compiler/qsc_formatter/src/formatter/tests.rs

```diff
@@ -0,0 +1,307 @@
+    check(
+        r#"
+    /// First
+/// Second
+    /// Third
+        namespace MyQuantumProgram {
+        open Microsoft.Quantum.Diagnostics;
+
+        @EntryPoint()
+        operation Main() : Int {
+            let x = 3;
+            let y = 4;
+
+            // Comment
+            return 5;
+        }
+            }
+"#,
+        &expect![[r#"
+            [
+                Edit {
+                    span: Span {
+                        lo: 0,
+                        hi: 5,
+                    },
+                    new_text: "",
+                },
+                Edit {
+                    span: Span {
+                        lo: 25,
+                        hi: 30,
+                    },
+                    new_text: "\n",
+                },
+                Edit {
+                    span: Span {
+                        lo: 39,
+                        hi: 48,
+                    },
+                    new_text: "\n",
+                },
+                Edit {
+                    span: Span {
+                        lo: 76,
+                        hi: 85,
+                    },
+                    new_text: "\n    ",
+                },
+                Edit {
+                    span: Span {
+                        lo: 120,
+                        hi: 130,
+                    },
+                    new_text: "\n\n    ",
+                },
+                Edit {
+                    span: Span {
+                        lo: 143,
+                        hi: 152,
+                    },
+                    new_text: "\n    ",
+                },
+                Edit {
+                    span: Span {
+                        lo: 176,
+                        hi: 189,
+                    },
+                    new_text: "\n        ",
+                },
+                Edit {
+                    span: Span {
+                        lo: 199,
+                        hi: 212,
+                    },
+                    new_text: "\n        ",
+                },
+                Edit {
+                    span: Span {
+                        lo: 222,
+                        hi: 236,
+                    },
+                    new_text: "\n\n        ",
+                },
+                Edit {
+                    span: Span {
+                        lo: 246,
+                        hi: 259,
+                    },
+                    new_text: "\n        ",
+                },
+                Edit {
+                    span: Span {
+                        lo: 268,
+                        hi: 277,
+                    },
+                    new_text: "\n    ",
+                },
+                Edit {
+                    span: Span {
+                        lo: 278,
+                        hi: 291,
+                    },
+                    new_text: "\n",
+                },
+            ]
+        "#]],
+    );
+}
+
+#[test]
+fn correct_empty_delimiters() {
```

Comment:
What does "correct" mean in this context?

Reply:
I've renamed this to `remove_spaces_between_empty_delimiters` for better clarity.